### PR TITLE
Fix build_man issues

### LINKED ISSUE #3364: build_man issues
```
(borg-env) tw@tux:~/w/borg$ python setup.py build_man
running build_man
building man pages (in docs/man)
building man page borg-benchmark(1)
building man page borg-benchmark-crud(1)
docs/virtmanpage.rst:56: (ERROR/3) Unexpected indentation.
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
building man page borgfs(1)
building man page borg-break-lock(1)
building man page borg-change-passphrase(1)
building man page borg-check(1)
building man page borg-create(1)
building man page borg-delete(1)
building man page borg-diff(1)
building man page borg-export-tar(1)
building man page borg-extract(1)
building man page borg-info(1)
building man page borg-init(1)
building man page borg-key(1)
building man page borg-key-change-passphrase(1)
building man page borg-key-export(1)
building man page borg-key-import(1)
building man page borg-key-migrate-to-repokey(1)
building man page borg-list(1)
building man page borg-mount(1)
building man page borg-prune(1)
docs/virtmanpage.rst:145: (ERROR/3) Unknown directive type "highlight".
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
.. highlight:: none

building man page borg-recreate(1)
building man page borg-rename(1)
building man page borg-serve(1)
building man page borg-umount(1)
building man page borg-upgrade(1)
building man page borg-with-lock(1)
building man page borg-patterns(1)
building man page borg-placeholders(1)
building man page borg-compression(1)
building man page borg(1)
docs/usage_general.rst.inc:: (WARNING/2) "table row spanning" not supported
docs/usage_general.rst.inc:: (WARNING/2) "table row spanning" not supported
docs/usage_general.rst.inc:: (WARNING/2) "table cell spanning" not supported
docs/usage_general.rst.inc:: (WARNING/2) "table row spanning" not supported
```

### PR DESCRIPTION
**Pull request by milkey-mouse**
Fixes #3364, except for the last warnings, which seem harmless anyway. Somebody could open an issue for sphinx as detailed [here](https://github.com/borgbackup/borg/issues/3364#issuecomment-346976483) to get rid of the last warnings if they were a little less lazy than me.

### CODE REVIEW

File: docs/usage/prune.rst [hunk 1]
```diff
@@ -34,6 +34,5 @@ first so you will see what it would do without it actually doing anything.
 
 There is also a visualized prune example in ``docs/misc/prune-example.txt``:
 
-.. highlight:: none
```
Comment (ThomasWaldmann) [inline docs/usage/prune.rst]: guess if you just kill that, it will apply ... the python highlighter (or bash?).

Comment (milkey-mouse) [reply docs/usage/prune.rst]: ah, of course. I only looked at the man page results and not at the html docs.

Comment (milkey-mouse) [reply docs/usage/prune.rst]: I've come to the conclusion I don't have enough experience with Sphinx to fix the highlight issue, at least the other minor fix can be merged

**Comment by ThomasWaldmann** [comment]
can you collapse the last 2 commits?

**Comment by milkey-mouse** [comment]
done

**Pull request merged**